### ISSUE TITLE
Convenção Classes

### ISSUE THREAD

**Issue by icavalheiro** [Treeofsavior/PortugueseTranslation#81]
Estava olhando a convenção dos nomes de classes e me deparei com a do Linker.

Tanto "Vinculador" como "Conector" são terriveis! Acredito que a melhor opção é manter "Linker" mesmo, igual o Highlander.

Pra quem não sabe do que estou falando: 
https://github.com/Treeofsavior/PortugueseTranslation/wiki/Tradu%C3%A7%C3%A3o-das-classes

**Comment by yalmeida69** [comment]
Eu sou a favor de traduzir todos os nomes que estão em inglês, inclusive Linker (para Vinculador) e Highlander (para Montanhês). Só que não são todos que concordam com isso. Tenho certeza que eventualmente vai ser feita outra votação sobre o assunto.

**Comment by Arkadel** [comment]
@icavalheiro É que houve uma votação no dia de ontem e Conector até agora é o preferido. Entretanto, essas votações deverão ser repetidas para grupos maiores de pessoas, para obtermos o aval de pelo menos boa parte das pessoas (e não ter gente demais preferindo o patch inglês por causa de um ou dois nomes que não gostaram ou não ficaram semelhantes ao original). Então, quanto mais gente saber para quando alguém levantar as novas votações, melhor.

**Comment by fabiohenm** [comment]
Os nomes não são definitivos. Pessoalmente gostei de Vinculador, mas veremos como fica no final.

**Comment by RicardoRFaria** [comment]
Não concordo de mantermos a palavra em inglês se temos uma tradução clara da mesma. E olha que eu particularmente acho a palavra em inglês muito mais bonita, um exemplo claro disso pra mim é "Sapper" que virou "Sapador".
É uma droga que nossa língua tenha uma sonoridade tão ruim para algumas coisas, mas temos que manter a mesma lógica de pensamento das outras classes, sem fazer concessões sem motivos.

**Comment by evtutchenko** [comment]
O problema de traduzir uma palavra de uma língua para outra é que as vezes ela se torna algo que as pessoas normalmente não usam.

No caso do Sapper ele poderia ser chamado de pioneiro ou engenheiro, todas são basicamente as mesmas funções. Sapador é muito bizarro por não ter nenhum uso na língua portuguesa brasileira.

**Comment by RicardoRFaria** [comment]
@evtutchenko "Sapador" tem um uso claro no exercito assim como "Sapper" no exército americano. Descobri isso no dia da discussão dessa tradução.
https://www.google.com.br/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=sapador

**Comment by evtutchenko** [comment]
@RicardoRFaria Verdade. Eu também vi isso depois. Foi equívoco meu. Mas ainda assim não é um termo que você escuta muito em qualquer lugar. Normalmente eu vejo "engenheiro" sendo usado quando falam dessa função. Lógico que dentro de um ambiente militar eu não tenho como falar, mas civilmente não é comum.

**Comment by RicardoRFaria** [comment]
@evtutchenko Mas a classe não se chama Engineering, e uma classe do tipo poderia vir a ser criada no futuro.
Não concordo com escolher outro nome se já temos a devida tradução, ninguém usa "Ladino" no dia a dia, provavelmente até menos que "Sapador" e nem por isso deixa de ser uma decisão clara para tradução de Rogue.
A palavra é horrível, concordo, mas não é sobre sonoridade né, é sobre tradução, e se já existe ela, é melhor não reinventar a roda.

**Comment by evtutchenko** [comment]
No dia-a-dia Ladino pode ser menos utilizado, mas virou um termo comum em rpgs, muitas pessoas sabem o que um Ladino é. Do mesmo jeito que ninguém fala loot o tempo todo fora de jogos.
(O certo seria "Engineer", "Engineering" significa engenharia).

Eu entendo sua posição. Mas traduzir para um termo que praticamente ninguém conhece vai dar no mesmo que deixar a palavra em inglês.
Eu acho que Pioneiro fica melhor; primeiro por ser uma palavra mais comum, segundo por tratar da mesma função.

**Comment by RicardoRFaria** [comment]
Ladino passou a ser conhecido do mundo do RPG, porquê alguém traduziu, se tivesse deixado Rogue porquê ninguém conhecia Ladino como você está propondo, nunca teria se tornado um termo conhecido.
Acredito que a principal preocupação é não ocupar termos que podem vir a ser usados no futuro e nos criar um problema. "Engineering" é certamente uma classe que pode surgir a qualquer momento e "Pioneiro" apesar de pouco provável, pode acontecer também.

**Comment by Arkadel** [comment]
Aliás, Pioneiro (Pioneer) é um termo que a IMC já usou bastante, e é realmente arriscado ter essa classe, ou mesmo essa colocação em personagens no futuro. Os Pioneer eram praticamente os "Revelator" do ToS em outro jogo famoso da IMC, o Granado Espada.

**Comment by evtutchenko** [comment]
Verdade, nesse caso realmente não tem como escapar.

**Comment by evtutchenko** [comment]
Mudando para outra classe.
Uma tradução melhor para Ranger não seria patrulheiro? E isso nada tem a ver com Game of Thrones. Eu tomo como base o table-top Tormenta (que é brasileiro) em que apesar deles não traduzirem a classe, eles sugerem um nome alternativo: Patrulheiro.
E para mim Sentinela e Patrulheiro tem sentidos parecidos porém ligeiramente diferentes. Um sentinela guarda um local fixo; o Patrulheiro uma área maior, já que ele ronda, vaga por aí.

Ranger vem do verbo "to range", e patrulhar também. Eu acredito que eles tenham sentidos mais próximos.

**Comment by yalmeida69** [comment]
Gostei de "Patrulheiro", acho um termo bem válido.

**Comment by Jigos** [comment]
Eu também acho que devíamos localizar todos os substantivos simples, como equipamentos, mapas e classes. Isso agrega valor a região e a caracteriza. Mesmo que soem estranho inicialmente, logo os jogadores irão se acostumar, principalmente os novatos. Acho patrulheiro válido para o Ranger, e Conector para o Linker está muito estranho, talvez encontremos um nome melhor que não seja uma tradução direta e literal, mas sim uma regionalização do termo.

**Comment by RicardoRFaria** [comment]
@Jigos concordo com tudo, salvo o trecho "mas sim uma regionalização do termo", segundo as diretrizes da IMC não devemos regionalizar, não é uma localização, apenas tradução.

**Comment by XTSJamal** [comment]
Estou fazendo a tradução de um trecho do ETC e me deparei com alguns nomes de classes gostaria a opinião de vocês para escolher o nome fixo para elas.

**Wizard:** Mago
**Pyromancer:** ??
**Cryomancer:** ??
**Psychokino:** ??
**Linker:** Vinculador
**Thaumaturge:** ??
**Elementalist:** ??
**Archer:** Arqueiro
**Ranger:** Patrulheiro
**Quarrel Shooter:** ??
**Sapper:** Sapador
**Hunter:** Caçador
**Wugushi:** ??
**Scout:** Batedor

**Comment by faicciruz** [comment]
https://github.com/Treeofsavior/PortugueseTranslation/wiki/Tradu%C3%A7%C3%A3o-das-classes

**Comment by faicciruz** [comment]
A tradução de Scout tá estranha na wiki.

**Comment by XTSJamal** [comment]
Otimo vi la no Wiki mesmo! E o batedor acho que condiz com o Scout.

**Comment by darkhershel** [comment]
ORANGER... o Ranger não se traduz... quem jogou D&D sabe.... Ranger é RANGER e nao "PATRULHEIRO"
Priest sacerdote... não tem como ser padre... já que father é como eles chamam de padre =) 
o highlander e o hoplite tbm nao se traduz não....

**Comment by evtutchenko** [comment]
Hoplite é hoplita, não tem motivo algum para não traduzir. E nem tradução é, apenas uma adaptação.
Não traduzir Highlander é frescura em minha opinião, já que montanhês é exatamente o que o nome significa. Esse nome não é dado exclusivamente aos escoceses que vivem no norte.
No caso do Ranger, "Patrulheiro" é a palavra que mais se assemelha ao sentido original.
Com "Priest" eu concordo, essa seria sim a melhor tradução

**Comment by darkhershel** [comment]
highlander é highlander... montanhes é montanhes .... é um titulo e não uma coisa... Se da o mesmo para o Ranger. Patrulheiro é uma coisa que muda totalmente td de ranger ....

**Comment by evtutchenko** [comment]
Highlander não é um título. É um nome dado à populações que vivem em terras altas/montanhas. Você tá prendendo este termo MUITO firmemente aos escoceses.

Sobre o Ranger eu concordo em parte, mas é o melhor termo em português para tradução.

**Comment by legendmasterr** [comment]
Ozzymandias, entendo seu ponto de vista, mas nesse caso a tradução não
teria sentido pois Highlander não é o nome dado à populações que vivem em
terras altas/montanhas, no jogo é o nome dado à profissão de seu personagem
sendo assim ou mantem o nome de Highlander ou teremos que criar um nome de
acordo com as habilidades dessa classe. Mesma coisa que as outras classes
no caso do Ranger o mais próximo seria patrulheiro mas ainda sim eu acho
muito vago para tradução dessa classe.



Em 27 de dezembro de 2015 16:15, Ozzymandias <notifications@github.com>
escreveu:

**Comment by Twerost** [comment]
Devo concordar que, por hora, o termo "Vinculador" se adeque mais à classe Linker, pois remete à função e da característica única dela.

Se houvesse algum sinônimo que remetesse ao ato de "aliar" ou "associar" personagens....

**Comment by evtutchenko** [comment]
@legendmasterr eu entendo isto.

Porém, não seria melhor entrar em contato com a IMC ou alguém que entenda coreano para verificar qual é o nome de cada classe em sua(jogo) língua original? Sendo nossa tradução baseada em outra tradução ela está vulnerável a equívocos de nossa parte.

**Comment by legendmasterr** [comment]
Ozzymandias, seria bom se tivéssemos como falar com alguém da IMC que
entenda o coreano e o português para que possamos tirar essa dúvida ficaria
mais fiel a ideia da profissão mas eu acho que eles não nos criticariam se
entrássemos em uma acordo para o nome da classe sem que seja traduzido ao
pá da letra eu fiz um teste por curiosidade e joguei o termo em coreano no
google tradutor e a tradução ficou " Highlander Mestre" existe a
possibilidade deles terem usado nomes em inglês mesmo porque quando fosse
adaptar para o inglês ficaria mais fácil.
A meu ver acho que poderíamos deixar o termo em inglês e só adaptar aqueles
que ficarem pejorativos.



Em 7 de janeiro de 2016 15:38, Ozzymandias <notifications@github.com>
escreveu:
